flocker/control/_model.py

```diff
@@ -301,9 +347,8 @@ def __invariant__(self):
         return (True, "")
 
     hostname = field(type=unicode, factory=unicode, mandatory=True)
-    applications = pset_field(Application)
-    manifestations = field(type=PMap, initial=pmap(), factory=pmap,
-                           mandatory=True)
+    applications = pset_field(Application, optional=True)
+    manifestations = pmap_field(unicode, Manifestation, optional=True)
```

Comment:
We will definitely want them on `NodeState`, but I was going to do that in followup branch.

It needs to be in `Node` because that's how current cluster state is communicated to agents, and the goal is to allow the container agent to say "oh, manifestation state is missing, I should wait until that's available".